views/reports/monthly_report.php

```diff
@@ -1594,5 +1651,25 @@ function initPage() {
 		display: none;
 	}
 }
+
+.loader {
+  border: 5px solid #f3f3f3;
+  border-radius: 50%;
+  border-top: 5px solid #3498db;
+  width: 35px;
+  height: 35px;
+  -webkit-animation: spin 2s linear infinite;
+  animation: spin 2s linear infinite;
+}
+
+@-webkit-keyframes spin {
```

Comment:
Hate to be a lame, but how well supported is `@keyframes` ? If we know we're ONLY using chrome I'm sure it's fine.

Reply:
Internet explorer 10, Chrome 4, safari 4, and Firefox 16 all support it, so I think we should be good. Only potential problem is if there using IE 9, but do we think they are?

Reply:
For right now if they are using out of date browsers then they should expect a lower user experience anyways. If anything we can worry about it if it actually becomes an issue.